papermill_origami/noteable_dagstermill/ops.py

```diff
@@ -289,116 +290,9 @@ def _t_fn(step_context, inputs):
     return _t_fn
 
 
-def define_noteable_dagstermill_solid(
```

Comment:
We no longer support the solids/pipelines API in dagster and we may remove the _legacy package at any time. I suggest removing this so you don't need to maintain compatibility with non-supported libraries and don't need to pin on an old version of dagster should we remove the _legacy package